app/lib/project/MetapolatorProject.js

```diff
@@ -91,6 +78,9 @@ define([
         this._log.addHandler(new log.Handler());
 
         this._momCache = Object.create(null);
+
+        this.chunk_io = {};
```

Comment:
You probably did not realize that we use the underscore to mark private interfaces. without an underscore the interface is regarded public. That means that the maintenance effort is very much bigger.

These new properties are not really public interfaces, are they? Also, please use camelCase.

I'm not yet sure if I like the implementation, so wait until the review is done.